src/axom/primal/operators/winding_number.hpp

```diff
@@ -197,142 +128,45 @@ int winding_number(const Point<T, 2>& R,
                    double edge_tol = 1e-8)
 {
   bool isOnEdge = false;
-  return winding_number(R, P, isOnEdge, includeBoundary, edge_tol);
+  return detail::polygon_winding_number(R, P, isOnEdge, includeBoundary, edge_tol);
 }
 
 /*!
- * \brief Computes the GWN for a 2D point wrt a 2D Bezier curve
+ * \brief Computes the GWN for a 2D point wrt a 2D NURBS curve
  *
  * \param [in] query The query point to test
```

Comment:
Consistency w/ function API

```suggestion
 * \param [in] q The query point to test
```